tests/unit/api/c/merger/test_dmx_merger.cpp

```diff
@@ -0,0 +1,1086 @@
+/******************************************************************************
```

Comment:
Very nice work on these tests.  Your mastery of unit testing is definitely improving.

Reply:
Thank you!

Reply:
I concur.  This is very nice!

Reply:
Thanks! :)